std_msgs/Header header        # Standard ROS header with timestamp
string device_id              # e.g., "MDL-012"
uint32 proto_version_hash
uint32 api_version_hash
uint32 firmware_version_hash
uint32 hdl_version_hash
